packages/contracts-bedrock/deploy-config/hardhat.json

```diff
@@ -55,5 +55,6 @@
   "proofMaturityDelaySeconds": 12,
   "disputeGameFinalityDelaySeconds": 6,
   "respectedGameType": 0,
-  "useFaultProofs": false
+  "useFaultProofs": false,
+  "fundDevAccounts": false
```

Comment:
If this is `true`, [this test](https://github.com/ethereum-optimism/optimism/blob/ab3a99ec1bd09a06fce5a0a2b07798eb8712947e/packages/contracts-bedrock/test/L2/SequencerFeeVault.t.sol#L143) fails. It seems like the address used for `sequencerFeeVault.RECIPIENT()` is a dev account and is funded so the balance assertion fails

Reply:
This could be updated in a follow up, it would be nice to have prefunded accounts for op-e2e next to use

Reply:
Tracking via [this issue](https://github.com/ethereum-optimism/interop/issues/58)